bindings/rust/s2n-tls/src/security.rs

```diff
@@ -84,12 +84,32 @@ pub const DEFAULT: Policy = policy!("default");
 /// <https://aws.github.io/s2n-tls/usage-guide/ch06-security-policies.html>
 pub const DEFAULT_TLS13: Policy = policy!("default_tls13");
 
+/// Default FIPS compliant policy
+///
+/// # Warning
+///
+/// Cipher suites, curves, signature algorithms, or other security policy options
+/// may be added or removed from "default_fips" in order to keep it up to date with
+/// current security best practices.
+///
+/// That means that updating the library may cause the policy to change. If peers
+/// are expected to be reasonably modern and support standard options, then this
+/// should not be a problem. But if peers rely on a deprecated option that is removed,
+/// they may be unable to connect.
+///
+/// If you instead need a static, versioned policy, choose one according to the s2n-tls usage guide:
+/// <https://aws.github.io/s2n-tls/usage-guide/ch06-security-policies.html>
+#[cfg(feature = "fips")]
+pub const DEFAULT_FIPS: Policy = policy!("default_fips");
```

Comment:
Ok good callout on not supporting tls1.3.

I debated if I should gate this or not. I choose to gate this because s2n-tls Rust customers really shouldn't be setting a DEFAULT_FIPS security policy since that gets set automatically in C.

Do you think its worth exposing it for general use or keeping it gated behind the feature flag?

Reply:
Planning to use `20230317` instead based on the [docs](https://aws.github.io/s2n-tls/usage-guide/ch06-security-policies.html)